design/decisions/AboutUs/Index.md

```diff
@@ -0,0 +1,30 @@
+﻿# About Us
```

Comment:
This content is mostly found already in the `/about/history/index.md` file, but the `/about/` content could be filled out more. Could you move this "Who we are" text over into `/about/index.md`? The goals section would best go into `/about/mission.md`.